layouts/partials/nav-item.html

```diff
@@ -1,29 +1,9 @@
 {{ $childLevel := add .Level 1 }}
-{{ $currentPage := .CurrentPage }}
+{{/*  {{ $currentPage := .CurrentPage }}  */}}
```

Comment:
Is this intentional to comment out, or are you still going to delete?

Reply:
No, I forgot to remove this. Will update. Thanks!

Reply:
Fixed